.gitignore

```diff
@@ -1,6 +1,9 @@
 # Retrieved 2017-Oct-12 from https://github.com/github/gitignore/blob/master/R.gitignore
 # Licensed under CC0-1.0 https://github.com/github/gitignore/blob/master/LICENSE
 
+# VSCode
+.vscode 
+
```

Comment:
Please move it to the end as I did for the changes for Material for MkDocs and Emacs.